js/Source/ImageResizer/ImageResizer.js

```diff
@@ -0,0 +1,84 @@
+/*
+---
+
+name: ImageResizer.js
+
+description: Resizes an image using a canvas, if supported.
+
+requires:
+ - Core/Options
+ - Core/Events
+
+provides: [ImageResizer]
+
+...
+*/
+
+(function(){
+
+  var ImageResizer = new Class({
```

Comment:
you have this inside a `(function(){})()` invocation, which means that the class isn't visible from outside this function. Made sense in the example I left as a comment because it also encapsulated the Delegator trigger. You can either ditch the anonymous function wrapper or do `window.ImageResizer` to make it visible outside the function. I'd do the former.